pages/qr.tsx

```diff
@@ -31,7 +30,12 @@ export default function QRCodePage() {
           undefined,
           videoEl.current
         )
-        window.location.href = appendUrlParams(result.getText())
+        const qrUrl = new URL(result.getText())
+        if (qrUrl.hostname === 'rcvr.app') {
```

Comment:
I don't think hard coding this here makes a lot of sense, does it?

Reply:
Do we have a config variable for this somewhere that I can use_

Reply:
There's an ENV variable. But I don't think there's a need for that. Simply check that it matches `location.hostname`.

Reply:
Ah, right. Perfect 🥳